### PR TITLE
Revamp Index Page

### LINKED ISSUE #270: [FEATURE] Create "real" index page and move the app itself to subroute
### Description

For SEO issues, we should move the page that is currently the index page to the subroute `/app`. It's caused by a redirect to the login page, in case people aren't logged in => Google indexes the content of the login page that isn't very relevant for people searching for it.

### Additional information

- migrate route
- update docs
- change Auth App-Redirect before deployment

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #270

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -5,6 +5,25 @@
...
+    --neutral-100: #22272e;
+    --neutral-200: #333a43;
+    --neutral-300: #555b66;
+    --neutral-400: #6e7379;
+    --neutral-500: #888f93;
+    --neutral-600: #a1a7ab;
+    --neutral-700: #babfbf;
+    --neutral-800: #d3d9dc;
+    --neutral-900: #f5f4f4;
+
```
Comment (Cahllagerfeld) [inline src/app.css:18]: add neutral colors to tailwind config

File: src/app.html [hunk 2]
```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="" />
```
Comment (eddiejaoude) [inline src/app.html:5]: Do we want to add a description here? It can come in another PR

File: tailwind.config.cjs [hunk 3]
```diff
@@ -4,6 +4,28 @@ const config = {
...
+          000: 'var(--neutral-000)',
+          100: 'var(--neutral-100)',
+          200: 'var(--neutral-200)',
+          300: 'var(--neutral-300)',
+          400: 'var(--neutral-400)',
+          500: 'var(--neutral-500)',
+          600: 'var(--neutral-600)',
+          700: 'var(--neutral-700)',
+          800: 'var(--neutral-800)',
+          900: 'var(--neutral-900)',
```
Comment (eddiejaoude) [inline tailwind.config.cjs:27]: Nice theme! 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: src/app.html (hunk 2 above)
Comment (Cahllagerfeld) [reply src/app.html:5]: It gets added with the seo component, and that tag would overwrite the one from the seo component, that's why it gets deleted here :D

**Pull request merged**